src/main/resources/public/js/pages/organization-search.js

```diff
@@ -31,36 +32,47 @@ class OrganizationSearch extends React.Component {
             organizationHistoryMessage: ''
         };
 
-        this.handleChange = this.handleChange.bind(this);
+        this._userOrganizationHistory = new UserOrganizationHistoryService();
     }
 
     componentDidMount() {
         this._handleOrganizationsHistory();
     }
 
-    _handleOrganizationsHistory = () => {
-        customFetch("/my/api/organizationHistory")
-            .then(res => {
-                if(res && res.length >0){
-                    this._sortOrganizationHistoryByDate(res);
-                    this.setState({organizationsHistory: res, organizationHistoryMessage: ''});
-                } else {
-                    this.setState({organizationHistoryMessage: i18n._(t`organization.search.history.empty`)});
-                }
-            });
+    _handleOrganizationsHistory = async () => {
+        try {
+            let res = await this._userOrganizationHistory.getOrganizationHistory();
+            if(res && res.length >0){
+                this._sortOrganizationHistoryByDate(res);
+                this.setState({organizationsHistory: res, organizationHistoryMessage: ''});
+            } else {
+                this.setState({organizationHistoryMessage: this.context.t("organization.search.history.empty")});
```

Comment:
isn't this the old way of translating messages ?

Reply:
WOW exactly, I miss focus...